src/npe2/manifest/_package_metadata.py

```diff
@@ -30,7 +30,9 @@ class PackageMetadata(BaseModel):
     class Config:
         extra = Extra.ignore
 
-    metadata_version: MetadataVersion = Field(
+    # allow str as a fallback in case the metata-version specification has been
+    # updated and we haven't updated the code yet
+    metadata_version: Union[MetadataVersion, str] = Field(
```

Comment:
Should we provide a validator that checks if this string is a valid version number?

Reply:
You can if you want to, but I'm not sure it matters (and could bring back the future proof problem).

Reply:
So why we just don't annotate with plain `str`?

Reply:
@Czaki, you can do whatever you want.  with my code above, I tried to indicate that normally this should be `MetadataVersion`, but yes we allow any string.  along with the provided comment it gives the reader a bit more context.  If you want to remove all that and make it just str, be my guest